### PR TITLE
Release audit

### PR DESCRIPTION
**Pull request by devongovett**

### CODE REVIEW

File: packages/@react-types/tabs/src/index.d.ts [hunk 1]
```diff
@@ -48,23 +48,29 @@ export interface AriaTabListProps<T> extends TabListProps<T>, DOMProps, AriaLabe
   isDisabled?: boolean
 }
 
+export interface AriaTabListProps<T> extends TabListProps<T>, AriaTabListBase, DOMProps, AriaLabelingProps {}
+
 export interface AriaTabPanelProps extends DOMProps, AriaLabelingProps {}
 
-interface SpectrumTabsProps<T> extends SingleSelection, DOMProps, StyleProps {
+interface SpectrumTabsProps<T> extends AriaTabListBase, SingleSelection, DOMProps, StyleProps {
```
Comment (snowystinger) [inline packages/@react-types/tabs/src/index.d.ts:55]: this needs to be exported

Comment (devongovett) [reply packages/@react-types/tabs/src/index.d.ts]: Good catch

**Review by dannify — APPROVED** [review]

File: packages/@react-types/tabs/src/index.d.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ export interface AriaTabProps {
 
 export interface TabListProps<T> extends CollectionBase<T>, SingleSelection {}
```
Comment (snowystinger) [inline packages/@react-types/tabs/src/index.d.ts:31]: FYI, @LFDanLu found https://github.com/adobe/react-spectrum/pull/1812/commits/8d53630440254a0318ab167e70277f6f9bdce589#diff-54fa76d32e9d408bcad2b8991e4a260e7e007c2917eff4a42bc393e88cbd9e01R31

File: packages/@react-types/tabs/src/index.d.ts [hunk 3]
```diff
@@ -48,23 +48,29 @@ export interface AriaTabListProps<T> extends TabListProps<T>, DOMProps, AriaLabe
...
 }
 
+export interface AriaTabListProps<T> extends TabListProps<T>, AriaTabListBase, DOMProps, AriaLabelingProps {}
+
 export interface AriaTabPanelProps extends DOMProps, AriaLabelingProps {}
 
-interface SpectrumTabsProps<T> extends SingleSelection, DOMProps, StyleProps {
+export interface SpectrumTabsProps<T> extends AriaTabListBase, SingleSelection, DOMProps, StyleProps {
+  /** The children of the <Tabs> element. Should include <TabList> and <TabPanels> elements. */
+  children: ReactNode,
```
Comment (snowystinger) [inline packages/@react-types/tabs/src/index.d.ts:57]: as well as https://github.com/adobe/react-spectrum/pull/1812/commits/8d53630440254a0318ab167e70277f6f9bdce589#diff-54fa76d32e9d408bcad2b8991e4a260e7e007c2917eff4a42bc393e88cbd9e01R57

**Review by dannify — APPROVED** [review]

**Pull request merged**